### PR TITLE
Show correct buy BSQ state on activation for take offer

### PR DESCRIPTION
**Pull request by ripcurlx**
Looks like this for take offer right now:
Bildschirmfoto 2022-03-28 um 12 45 41
But should look like this:
Bildschirmfoto 2022-03-28 um 12 30 12

It only works for create offer right now as intended.

### CODE REVIEW

**Comment by ripcurlx** [comment]
@jmacxx Could you please ACK this? Thanks!

File: desktop/src/main/java/bisq/desktop/main/offer/bisq_v1/takeoffer/TakeOfferView.java [hunk 1]
```diff
@@ -336,11 +336,9 @@ protected void activate() {
             tradeFeeInBsqToggle.setManaged(false);
             buyBsqBox.setVisible(false);
             buyBsqBox.setManaged(false);
-        }
-
-        if (!model.isShowBuyBsqHint()) {
```
Comment (ghost) [inline desktop/src/main/java/bisq/desktop/main/offer/bisq_v1/takeoffer/TakeOfferView.java:341]: I see similar code fragment in [MutableOfferView](https://github.com/bisq-network/bisq/blob/00dc7998ef3e77496605def8111eae8a91b7faae/desktop/src/main/java/bisq/desktop/main/offer/bisq_v1/MutableOfferView.java#L277-L280) - shouldn't it be updated similarly?

Comment (ripcurlx) [reply desktop/src/main/java/bisq/desktop/main/offer/bisq_v1/takeoffer/TakeOfferView.java:341]: It works there as the listener is triggered, but yes I guess it is better to have the same code there as well. I'll push an additional commit.

**Review by ghost — APPROVED** [review]
utACK

**Review by ghost — APPROVED** [review]
ACK

image

**Review by bisq-github-admin-3 — APPROVED** [review]
utACK - based on https://github.com/bisq-network/bisq/pull/6121#pullrequestreview-923476350

**Pull request merged**